src/app/desktop/components/Instances/Instance.js

```diff
@@ -144,6 +144,28 @@ const Instance = ({ instanceName }) => {
 
   const isPlaying = startedInstances[instanceName];
 
+  const calcTime = minutes => {
+    const days = Math.floor(minutes / 1440); // 60*24
+    const hours = Math.floor((minutes - days * 1440) / 60);
+    const min = Math.round(minutes % 60);
+    const weeks = Math.floor(days / 7);
+    const months = Math.floor(weeks / 4);
+
+    if (days < 7) {
+      if (days > 0) {
+        return days + ' d, ' + hours + ' h, ' + min + ' m';
+      } else {
+        if (hours > 0) {
+          return hours + ' h, ' + min + ' m';
+        } else return min + ' minutes';
+      }
+    } else {
+      if (months > 0) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected if as the only statement in an else block.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5285888)